jazz_test-lambda/components/utils.js

```diff
@@ -0,0 +1,49 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+    Helper functions for Events
+    @module: utils.js
+    @author:
+    @version: 1.0
+**/
+
+var validateARN = (arn) => {
+    if (arn) {
```

Comment:
another way to validate arn(cleaner), this can be made as a utility. you can do this as a separate PR as well as in improvement

function arnValidator(arn) {
const arnProperties = [
    'arn',
    'aws',
    'service',
    'region',
    'account',
    'name'
]

return  arn.split(':').reduce((result, item, index) => {
        result[ arnProperties[index] ] = item;
        return result;
    }, {});
}

and then use it to validate the output of the function.

Reply:
I would need more clarity on this one ... 
can we move ahead with the existing method for now and cover it as an enhancement ??

Reply:
yes we can talk offline but we can definitely do this as a separate PR too.